src/interactive/Modal.jsx

```diff
@@ -129,6 +133,19 @@ class Modal extends React.Component {
 			</div>
 		);
 
+		const closeArea = (
+			<div className='padding--all'>
+				<Button onClick={this.onDismiss} className={dismissButtonClasses}>
+					<Icon shape='cross' size='s' />
+				</Button>
+			</div>
+		);
+
+		const heroStyles = {
+			backgroundColor: heroBgColor || 'transparent',
+			backgroundImage: `url(${heroBgImage})`
```

Comment:
When `heroBgImage` isn't set, `background-image: url(undefined)` gets put in the markup. Is there a neater way?